syntax/soy.tmLanguage.json

```diff
@@ -95,7 +95,7 @@
 							}
 						},
 						{
-							"match": "\\b(deltemplate|template|element|call|delcall|dynacall|namespace|delpackage|package)\\b(?:\\s+([\\w\\d.]+))?",
```

Comment:
Thank you for your PR! It would be nice to keep the `delpackage` as well for backward compatibility for older projects.